ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -5523,3 +5523,111 @@ RFVSC2X:
   - return_fan_speed_percentage_command_2
   implements:
   - OPERATIONAL
+
+#New abstracts for DE-FRA-NEM32
+
+EXTOR:
+  description: "shade control with extension monitoring based on outside illuminance"
+  is_abstract: true
+  uses:
+  - shade_extent_percentage_command
+  - outside_illuminance_sensor
+  - min_illuminance_setpoint
+  - max_illuminance_setpoint
+
+DPHC2:
+  description: "Two-pipe heating and cooling control for radiant panels"
+  is_abstract: true
+  opt_uses:
+  - supply_water_isolation_valve_status
+  - return_water_isolation_valve_status
+  - supply_water_temperature_setpoint
+  uses:
+  - supply_water_temperature_sensor
+  - return_water_temperature_sensor
+  - water_riser_mode
+
+DPHC4:
+  description: "Four pipe heating and cooling controls for radiant panels"
+  is_abstract: true
+  opt_uses:
+  - supply_water_isolation_valve_status
+  - return_water_isolation_valve_status
+  - heating_supply_water_temperature_setpoint
+  - chilled_supply_water_temperature_setpoint
+  uses:
+  - heating_supply_water_temperature_sensor
+  - heating_return_water_temperature_sensor
+  - chilled_supply_water_temperature_sensor
+  - chilled_return_water_temperature_sensor
+  - water_riser_mode
```

Comment:
Its almost the same situation as mentioned above. These RP's are connected to a four pipe installation by using a 6-way valve. Therefore they don't need the water riser mode because they can always either cool or heat (I removed it...).
But we have the situation regarding the valve. No clear 0-100% command but two loads (0-100% cooling or 0-100% heating)